lib/ims/lti/models.rb

```diff
@@ -31,5 +31,10 @@ module Models
     require_relative 'models/content_item_placement'
     require_relative 'models/content_item_container'
     require_relative 'models/image'
+    require_relative 'models/agent'
```

Comment:
we should probably namespace the membership service models in the off chance IMs creates conflicting models in the future